src/value.rs

```diff
@@ -132,3 +141,33 @@ pub fn type_from_str(name: &str) -> Option<RbxValueType> {
         _ => None,
     }
 }
+
+struct Color3Value {
```

Comment:
We should be able to derive `Debug`, `Clone`, and `Copy` on this type to make our lives easier elsewhere.